libs/webb-ui-components/src/components/Socials/Socials.tsx

```diff
@@ -0,0 +1,22 @@
+import { SocialsProps } from './types';
+import { socialConfigs } from '../../constants';
+
+export const Socials = (props: SocialsProps) => {
+  const { iconPlacement = 'start' } = props;
+
+  const className = `flex items-center justify-${iconPlacement} space-x-4`;
```

Comment:
The `justify-${iconPlacement}` won't work because TailwindCSS cannot parse the className, please check [this](https://tailwindcss.com/docs/content-configuration#dynamic-class-names) out @vutuanlinh2k2.